Small update for `Beatmap/Beatmap description` (#5988)
ppy/osu-wiki: wiki/Beatmap/Beatmap_description/en.md

# Beatmap description

**Beatmap description** is a small area under a [beatmap](/wiki/Beatmap)'s page that gives players information about the beatmap using [BBCode](/wiki/BBCode). It usually highlights some facts about the beatmap its creator deemed important enough, or is used to advertise other mapping projects.

## Uses and information

Popular descriptions of [ranked](/wiki/Beatmap/Category#ranked) beatmaps often include the following:

- A list of all the [guest difficulties](/wiki/Beatmap/Guest_difficulty) and their creators
- Information that would help a player decide whether they want to download the beatmap or not
- A list of edits, or changelog

### Images

> Many beatmap descriptions contain at least one image (often an animated [GIF](https://en.wikipedia.org/wiki/GIF "Wikipedia") of some sort) which is used to express and illustrate either the creator, song, and/or beatmap in a flashy way. They can also take places of a title or header to add flair and quality. To learn how to add an image using BBCode, check out the [BBCode article's tutorial on images](/wiki/BBCode#image).

Reviewer: ```suggestion
Many beatmap descriptions contain at least one image (often an animated [GIF](https://en.wikipedia.org/wiki/GIF "Wikipedia") of some sort) which is used to express and illustrate either the creator, song, and/or beatmap in a flashy way. They can also serve as a title or header to add flair and quality. To learn how to add an image, check out the [BBCode article's section on images](/wiki/BBCode#image).
```

### Difficulty sections

> A "difficulty section" is a list of all the guest difficulties and their creators located in a beatmap description. It is mostly used to credit the guest mappers and showcase the difficulties of a beatmap. Difficulty sections usually have images for the icons.

Reviewer: ```suggestion
A "difficulty section" is a list of all guest difficulties and their creators located in a beatmap description. It is mostly used to credit the guest mappers and showcase the difficulties of a beatmap. Difficulty sections usually use icons to mark the difficulty levels.
```

Author: ```suggestion
A _difficulty section_ is a list of all guest difficulties (often alongside with the respective icons) and their creators located in a beatmap description. It is mostly used to credit the guest mappers.
```
